control_panel_api/tests/test_views.py

```diff
@@ -160,13 +160,11 @@ def test_create_when_name_invalid(self):
         response = self.client.post(reverse('s3bucket-list'), data)
         self.assertEqual(HTTP_400_BAD_REQUEST, response.status_code)
 
-    @patch('control_panel_api.services.create_bucket')
-    @patch('control_panel_api.services.create_bucket_policies')
-    def test_create_calls_apis(self, mock_create_bucket, mock_create_bucket_policies):
+    @patch('control_panel_api.services.bucket_create')
+    def test_create_calls_apis(self, mock_bucket_create):
         data = {'name': 'test-bucket-123'}
         self.client.post(reverse('s3bucket-list'), data)
-        mock_create_bucket.assert_called()
-        mock_create_bucket_policies.assert_called()
+        mock_bucket_create.assert_called()
```

Comment:
I think I prefer this more, the `services` layer is now abstracting out the "complexity" of creating an S3 bucket and related access policies.